core/src/test/java/co/infinum/goldfinger/ValidateUtilsTest.java

```diff
@@ -50,7 +51,7 @@ public void auth_valid() {
             .negativeButtonText(NEGATIVE_BUTTON_TEXT)
             .description(DESCRIPTION)
             .subtitle(SUBTITLE)
-            .deviceCredentialsAllowed(true)
+            .allowedAuthenticators(BiometricManager.Authenticators.BIOMETRIC_WEAK & BiometricManager.Authenticators.DEVICE_CREDENTIAL)
```

Comment:
Same here. Im aware this is just a test, but maybe we should add some docs to this method to make sure that no one makes this mistake.